packages/analytics/analytics-utils/README.md

```diff
@@ -0,0 +1,19 @@
+# @kong-ui-public/analytics-utils
+
+{A description of this package}
+
+- [Features](#features)
+- [Usage](#usage)
+  - [Install](#install)
+
+## Features
+
+- Common types and utils for kong analytics
+
+## Usage
+- Install and import/use types and utils as needed
+### Install
+
+```
+  yarn add @kong-ui-public/analytics-utils
+```
```

Comment:
issue: with this being a public repository, you need to complete a README for the existing functionality. Please see the existing README files for the template/example

Reply:
What do you mean existing functionality? It's just a bunch of interfaces/types right now I don't think that's applicable yet?

Reply:
Sorry, I typed too fast. Can you please at least fill out that description placeholder at the top, and add a usage example.

Reply:
Added